game-headless/src/main/java/org/triplea/game/server/HeadlessGameRunner.java

```diff
@@ -17,6 +26,20 @@ public static void main(final String[] args) {
   }
 
   private static Injections constructInjections() {
-    return Injections.builder().engineVersion(new ProductVersionReader().getVersion()).build();
+    return Injections.builder()
+        .engineVersion(new ProductVersionReader().getVersion())
+        .playerTypes(gatherPlayerTypes())
+        .build();
+  }
+
+  private static Collection<PlayerTypes.Type> gatherPlayerTypes() {
+    return Stream.of(
+            PlayerTypes.getBuiltInPlayerTypes(),
+            List.of(new PlayerTypes.AiType(new DoesNothingAiProvider())),
```

Comment:
Bots are not able to use AI players, does this make sense to have?

Reply:
I don't know.  The `DoesNothingAi` existed in the Headless game before this change so I'm not adding anything new.  I could remove it but I'm not sure how to test to ensure that it isn't actually used.  I do know that `PlayerSelectorRow#setDefaultPlayerType` requires the `DoesNothingAi`.  Is that method ever called in the Headless code paths?

Reply:
Let's see if this is really needed before merging.

Reply:
Second question, do bots really need a selection of player types? Isn't this to support the launch screen? If this is needed, are bots broken currently without this?

Reply:
Bots are currently broken as I mentioned in the testing section.

I removed the does nothing ai line and started a bot.  It looks like it isn't complaining.  I can actually start a game now.